app/controllers/concerns/project_concern.rb

```diff
@@ -0,0 +1,13 @@
+module ProjectConcern
+  extend ActiveSupport::Concern
+
+  private
+
+  def change_date(project)
```

Comment:
That said, this method works fine too, so whatever @hooverlunch says is fine with me.